cmd/hermes/cmd_eth.go

```diff
@@ -213,6 +248,114 @@ var cmdEthFlags = []cli.Flag{
+		EnvVars:     []string{"HERMES_ETH_SUBNET_ATTESTATION_COUNT"},
+		Usage:       "Number of random attestation subnets to select when type=random",
+		Value:       ethConfig.SubnetAttestationCount,
+		Destination: &ethConfig.SubnetAttestationCount,
+	},
+	&cli.Uint64Flag{
+		Name:        "subnet.attestation.start",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_ATTESTATION_START"},
+		Usage:       "Start of subnet range (inclusive) for attestation when type=static_range",
+		Value:       ethConfig.SubnetAttestationStart,
+		Destination: &ethConfig.SubnetAttestationStart,
+	},
+	&cli.Uint64Flag{
+		Name:        "subnet.attestation.end",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_ATTESTATION_END"},
+		Usage:       "End of subnet range (exclusive) for attestation when type=static_range",
+		Value:       ethConfig.SubnetAttestationEnd,
+		Destination: &ethConfig.SubnetAttestationEnd,
+	},
+	// Subnet flags for sync committee.
+	&cli.StringFlag{
+		Name:        "subnet.synccommittee.type",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_SYNCCOMMITTEE_TYPE"},
+		Usage:       "Subnet selection strategy for sync committee topics (all, static, random, static_range)",
+		Value:       ethConfig.SubnetSyncCommitteeType,
+		Destination: &ethConfig.SubnetSyncCommitteeType,
+	},
+	&cli.StringFlag{
+		Name:        "subnet.synccommittee.subnets",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_SYNCCOMMITTEE_SUBNETS"},
+		Usage:       "Comma-separated list of subnet IDs for sync committee when type=static",
+		Value:       ethConfig.SubnetSyncCommitteeSubnets,
+		Destination: &ethConfig.SubnetSyncCommitteeSubnets,
+	},
+	&cli.Uint64Flag{
+		Name:        "subnet.synccommittee.count",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_SYNCCOMMITTEE_COUNT"},
+		Usage:       "Number of random sync committee subnets to select when type=random",
+		Value:       ethConfig.SubnetSyncCommitteeCount,
+		Destination: &ethConfig.SubnetSyncCommitteeCount,
+	},
+	&cli.Uint64Flag{
+		Name:        "subnet.synccommittee.start",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_SYNCCOMMITTEE_START"},
+		Usage:       "Start of subnet range (inclusive) for sync committee when type=static_range",
+		Value:       ethConfig.SubnetSyncCommitteeStart,
+		Destination: &ethConfig.SubnetSyncCommitteeStart,
+	},
+	&cli.Uint64Flag{
+		Name:        "subnet.synccommittee.end",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_SYNCCOMMITTEE_END"},
+		Usage:       "End of subnet range (exclusive) for sync committee when type=static_range",
+		Value:       ethConfig.SubnetSyncCommitteeEnd,
+		Destination: &ethConfig.SubnetSyncCommitteeEnd,
+	},
+	// Subnet flags for blob sidecar.
+	&cli.StringFlag{
+		Name:        "subnet.blobsidecar.type",
+		EnvVars:     []string{"HERMES_ETH_SUBNET_BLOBSIDECAR_TYPE"},
+		Usage:       "Subnet selection strategy for blob sidecar topics (all, static, random, static_range)",
+		Value:       ethConfig.SubnetBlobSidecarType,
+		Destination: &ethConfig.SubnetBlobSidecarType,
+	},
+	&cli.StringFlag{
```

Comment:
there are actually the three convenience flags:

- `Int64SliceFlag`
- `IntSliceFlag`
- `StringSliceFlag`

that abstract away parsing multiple values for a single flag. This could make `parseSubnetList` obsolete.

Reply:
Thanks @dennis-tra, that's a good idea. Change made, [0e81985](https://github.com/probe-lab/hermes/pull/51/commits/0e8198573c45308b2373e985211d6c769afd5eaa).